pkg/controllers/deprovisioning/events.go

```diff
@@ -26,8 +26,8 @@ func TerminatingNode(node *v1.Node, reason string) events.Event {
 	return events.Event{
 		InvolvedObject: node,
 		Type:           v1.EventTypeNormal,
-		Reason:         "ConsolidateTerminatingNode",
-		Message:        fmt.Sprintf("Consolidating node via %s", reason),
+		Reason:         "DeprovisioningTerminatingNode",
```

Comment:
Do we need to prefix all of these with Deprovisioning?

Reply:
This was from a previous change. Not changing anything right now.